### PR TITLE
115 dashboard implementation

### PR DESCRIPTION
**Pull request by STom6**
Screenshot 2024-02-26 at 21 00 51

### CODE REVIEW

File: src/components/dashboard/widgets/OverView/FileItem.vue [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    </v-col>
+    <v-col cols="1">
+      <span :data-cy="$t(`overviewwidget-fileitem-tidyscore`)">
+        {{ getGrade(file.tidy_score) }}
+      </span>
+    </v-col>
+    <v-col cols="1">
+      <img
+        src="@/assets/icons/redirect.svg"
+        style="cursor: pointer; width: 30px; height: 30px; margin-top: -4px"
```
Comment (EmileEttel) [inline src/components/dashboard/widgets/OverView/FileItem.vue:16]: Put this in a class

File: src/components/dashboard/widgets/OverView/OverViewWidget.vue [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+        :value="item.value"
+        :class="{ 'active-tab': activeTab === item.value }"
+        :data-cy="$t(`overviewwidget-tabs-${item.value}`)"
+        @click="changeTab(item.value)"
+      >
+        {{ $t(`dashboard.widgets.overView.title.${item.label}`) }}
+      </v-tab>
+    </v-tabs>
+    <!-- <ListFile :tidy-hub-api="tidyHubApi + '/proxy/get_files?amount=10&sort_by=size'" :tab="selectedTabLabel" /> -->
+    <ListFile :tidy-hub-api="tidyHubApi + '/Gateway/getWidgetOverView'" :tab="selectedTabLabel" />
```
Comment (EmileEttel) [inline src/components/dashboard/widgets/OverView/OverViewWidget.vue:16]: Why is this the only one that does do the tidyHubApi + thing here ?

File: src/components/dashboard/widgets/OverView/OverViewWidget.vue [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+        :key="index"
+        :value="item.value"
+        :class="{ 'active-tab': activeTab === item.value }"
+        :data-cy="$t(`overviewwidget-tabs-${item.value}`)"
+        @click="changeTab(item.value)"
+      >
+        {{ $t(`dashboard.widgets.overView.title.${item.label}`) }}
+      </v-tab>
+    </v-tabs>
+    <!-- <ListFile :tidy-hub-api="tidyHubApi + '/proxy/get_files?amount=10&sort_by=size'" :tab="selectedTabLabel" /> -->
```
Comment (EmileEttel) [inline src/components/dashboard/widgets/OverView/OverViewWidget.vue:15]: Remove comment ?

File: src/components/dashboard/widgets/OverView/TidyScore.vue [hunk 4]
```diff
@@ -2,6 +2,7 @@
   <div class="TidyScore">
     <div class="text-center">
       <img
+        :data-cy="$t('tidyscore-grade-img')"
         :src="getGradeSVGPath()"
         alt="SVG Image"
         style="width: 50px; height: 50px"
```
Comment (EmileEttel) [inline src/components/dashboard/widgets/OverView/TidyScore.vue:8]: class ?

File: src/components/dashboard/widgets/OverView/TidyScore.vue [hunk 5]
```diff
@@ -10,43 +11,50 @@
     </div>
     <p>
       <strong
+        :data-cy="$t('tidyscore-file-name')"
         style="display: flex; flex-direction: column; align-items: center"
```
Comment (EmileEttel) [inline src/components/dashboard/widgets/OverView/TidyScore.vue:15]: class ?

File: src/components/dashboard/widgets/OverView/TidyScore.vue [hunk 6]
```diff
@@ -10,43 +11,50 @@
...
+          :data-cy="$t(`tidyscore-${value}`)"
+        >
           <div v-if="value">
             <strong>{{ $t(`fileItem.${key}`) }}</strong>
             <img
-              src="../../assets/false.svg"
+              :data-cy="$t(`tidyscore-${value}-false`)"
+              src="@/assets/icons/false.svg"
               alt="False Icon"
               style="width: 20px; height: 20px"
```
Comment (EmileEttel) [inline src/components/dashboard/widgets/OverView/TidyScore.vue:39]: class ?

File: src/components/dashboard/widgets/OverView/TidyScore.vue [hunk 7]
```diff
@@ -10,43 +11,50 @@
...
             />
           </div>
           <div v-else>
             <strong>{{ $t(`fileItem.${key}`) }}</strong>
             <img
-              src="../../assets/true.svg"
+              :data-cy="$t(`tidyscore-${value}-true`)"
+              src="@/assets/icons/true.svg"
               alt="True Icon"
               style="width: 20px; height: 20px"
```
Comment (EmileEttel) [inline src/components/dashboard/widgets/OverView/TidyScore.vue:48]: class ?

File: src/assets/translation/en.json [hunk 8]
```diff
@@ -0,0 +1,83 @@
...
+        "About" : "About"
+    },
+    "help": {
+        "helpText1" : "This is where I would put the tutorial for tidyBee",
+        "helpText2" : "Once its written"
+    },
+    "fileItem" : {
+        "lastUsed" : "Last Used: ",
+        "misnamed" : "Misnamed: ",
+        "heavy " : "Heavyest: ",
```
[the remark this answers is not in the document]
Comment (STom6) [reply src/assets/translation/en.json:19]: here we want to know if a file is heavy, in relation to the defined rule. So here we want to know if the file is judged as "Heavy" and not the "Heaviest".

**Review by EmileEttel — APPROVED** [review]
LGTM

**Review by EmileEttel — APPROVED** [review]
Still LGTM

**Review by majent4 — APPROVED** [review]

**Pull request merged**